File: wrapper/src/periodic_table.cc
#include "rust/cxx.h"
#include <rdkit/GraphMol/PeriodicTable.h>

namespace RDKit {
  const std::vector<int> & get_valence_list(unsigned int atomic_number) {
    PeriodicTable *pt = RDKit::PeriodicTable::getTable();
    return pt->getValenceList(atomic_number);
  }
}

File: src/bridge/periodic_table.rs
#[cxx::bridge(namespace = "RDKit")]
pub mod ffi {
    unsafe extern "C++" {
        include!("wrapper/include/periodic_table.h");

        pub fn get_valence_list(atomic_number: u32) -> &'static CxxVector<i32>;
    }
}
